projects/extension/src/background/ConnectionManager.ts

```diff
@@ -34,6 +34,10 @@ export class ConnectionManager extends (EventEmitter as { new(): StateEmitter })
     return this.#smoldots.map(s => s.name);
   }
 
+  get apps(): AppMediator[] {
```

Comment:
why did you create a new way of iterating the apps and networks instead of fixing the `getState()` method to add the extra data you needed?

Reply:
Didn't realise that getState is there. I will give it a try - though as per previous comment I tried it but I did not get the apps that were registered (0 were always returned). We can actually have a look on this also on Monday